osu.Game.Rulesets.Catch/Skinning/LegacyComboCounter.cs

```diff
@@ -0,0 +1,156 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using osu.Framework.Allocation;
+using osu.Framework.Bindables;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Game.Graphics.Sprites;
+using osu.Game.Graphics.UserInterface;
+using osu.Game.Rulesets.Catch.UI;
+using osu.Game.Screens.Play;
+using osu.Game.Skinning;
+using osuTK;
+using osuTK.Graphics;
+
+namespace osu.Game.Rulesets.Catch.Skinning
+{
+    internal class LegacyComboCounter : CompositeDrawable, ICatchComboCounter
```

Comment:
if this is in the ruleset assembly already does it make much difference whether it's `public` or `internal`? would just make `public` as we're trying to avoid internals for dynamic compilation reasons

Reply:
had the habit of marking any legacy class internal, but that's an absolutely good point, will change it back to public.